import/xsl/dspace-xoai.xsl

```diff
@@ -0,0 +1,164 @@
+<!-- 
+    The contents of this file are subject to the license and copyright
+    detailed in the LICENSE and NOTICE files at the root of the source
+    tree and available online at
+    http://www.dspace.org/license/
+	Developed by DSpace @ Lyncode <dspace@lyncode.com>
```

Comment:
Is this header comment accurate? Seems like a DSpace-specific comment that no longer applies in the context of VuFind (e.g. we don't have a NOTICE file). Does this need to be revised to indicate that it is derived from something created by Lyncode, rather than to say that it is created by Lyncode?

Reply:
Also, http://www.dspace.com/license is a dead link.

Reply:
thanks @demiankatz I have replaced this with a header comment more inline with the one in dspace-dim.xsl ( https://github.com/vufind-org/vufind/blob/dev/import/xsl/dspace-dim.xsl )